### ISSUE TITLE
a11y: Done button breaks focus order for new subjects

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#7274]
>Now that this has merged to production, pressing Done also focuses the task container, skipping past the new subject and failing [WCAG SC 2.4.3 Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html), which is a level A failure. 
>
>In a screen reader, there's no indication that a new subject has loaded, or that you need to navigate backwards through the page in order to listen to it. After submitting a classification, keyboard focus should probably jump to the new subject. 
>
>This is going to make audio subjects and sonification projects very difficult for a blind/visually-impaired audience, which is a regression from PFE.

_Originally posted by @eatyourgreens in https://github.com/zooniverse/front-end-monorepo/issues/7218#issuecomment-3937778093_
            
Since #7218 merged, pressing Done now focuses the task area for a new subject. This has a few negative accessibility/usability implications:
- **screen readers:** sending a BVI volunteer back to the beginning of the workflow tasks form, after submitting a classification, gives the impression that they've made a mistake and need to fill out their classification again. They won't know that a new subject has loaded, unless they navigate backwards from the task to the subject (which is unexpected.) NB. we've had similar feedback in the past (about having to unexpectedly navigate backwards) from a blind volunteer who used NVDA to participate in audio projects.
- **screen magnifiers:** similarly, screen magnifiers jump back to the beginning of the workflow after submitting a classification, creating the impression that something went wrong and the workflow needs to be filled out again.
- **small screens:** phone screens have the same problem as desktop magnifiers. Done scrolls to the top of the task area, not the top of the classifier, so a volunteer might miss that the subject has changed. You can reproduce this by classifying a few SquirrelMapper photos on a phone. 

After submitting a classification, keyboard focus should probably shift to the new subject to satisfy [WCAG SC 2.4.3 Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html).

At the moment, the task area focus management fails WCAG level A. Project owners like the British Library, National Archives or Royal Museums Greenwich (all funded by DCMS) [require WCAG AA](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps#meeting-accessibility-requirements).

**Comment by eatyourgreens** [comment]
Here's a video showing this bug on Comet Catchers. After each classification, focus shifts back to the task container and the subject is pushed off the top of the screen. 

https://github.com/user-attachments/assets/e15e6fa1-cecf-43d3-a823-f26c35b29d05
